examples/example_write_individual_address.py

```diff
@@ -0,0 +1,21 @@
+"""Example for writing an individual address to a KNX device."""
+import asyncio
+
+from xknx import XKNX
+from xknx.management import procedures
+from xknx.telegram import IndividualAddress
+
+
+async def main():
+    """
+    Write the individual address 1.1.3 to a device in programming mode.
+
+    This fails if multiple devices are in programming mode and/or when there is no device found in programming mode.
+    """
+    async with XKNX() as xknx:
+        individual_address = IndividualAddress("1.1.3")
```

Comment:
The example can be made more useful if the individual address can be passed as an argument, see https://github.com/XKNX/xknx/blob/88fe96ba54f5d9da3fb1dab134d74b0fb0538dff/examples/example_restart.py#L16 as an example.